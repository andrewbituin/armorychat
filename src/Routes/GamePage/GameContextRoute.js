import React from 'react';
import { GameProvider } from "../../Contexts/gameContext";
import GamePage from './GamePage';

export default function GameContextRoute(props) {
  return (
    <GameProvider>
      <GamePage {...props}/>
    </GameProvider>
  )
}