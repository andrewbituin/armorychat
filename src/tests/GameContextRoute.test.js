import React from 'react';
import GameContextRoute from '../Routes/GamePage/GameContextRoute';
import { MemoryRouter } from 'react-router-dom';
import renderer from "react-test-renderer";
import { shallow } from 'enzyme';

describe('GameContextRoute component', () => {

  it('renders without crashing', () => {
    shallow(<GameContextRoute />)
  })

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <GameContextRoute />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})