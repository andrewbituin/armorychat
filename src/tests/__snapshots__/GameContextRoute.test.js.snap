// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameContextRoute component renders the UI as expected 1`] = `
<div>
  Loading...
</div>
`;
